src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/SanitizerInfo.cs

```diff
@@ -65,5 +65,10 @@ public bool Equals(SanitizerInfo other)
                 && this.SanitizingMethods == other.SanitizingMethods
                 && this.SanitizingInstanceMethods == other.SanitizingInstanceMethods;
         }
+
+        public bool RequiresValueContentAnalysis()
+        {
```

Comment:
This could be a property, no?  Move it above with the other properties, and add /// <xml> comments.